pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -105,17 +105,16 @@ async def acker(subscription: str,
                     ack_queue.task_done()
             except aiohttp.client_exceptions.ClientResponseError as e:
                 if e.status == 400:
-                    log.error(
-                        'Ack error is unrecoverable, '
-                        'one or more messages may be dropped', exc_info=e)
+                    log.exception('unrecoverable ack error, one or more '
+                                  'messages may be dropped: %s', e)
 
                     async def maybe_ack(ack_id: str) -> None:
                         try:
                             await subscriber_client.acknowledge(
                                 subscription,
                                 ack_ids=[ack_id])
                         except Exception as ex:
-                            log.warning('Ack failed for ack_id=%s', ack_id,
+                            log.warning('ack failed', extra={'ack_id': ack_id},
```

Comment:
Looks like tests are using `caplog` to grab the specific message, will have to be updated.